RELEASE_NOTES.md

```diff
@@ -0,0 +1,147 @@
+# BEAM Release Notes
+## 1.0.0
+### New Features
+- DemandFollowingRepositionig: Reposition the random activity's location.
+- Added CSV and XML writer for PhysSim events.
```

Comment:
I guess xml was always there? Or do you mean allow switching between csv and XML?